snuba_sdk/visitors.py

```diff
@@ -122,8 +122,9 @@ def _visit_totals(self, totals: bool) -> TVisited:
 
 
 class Translation(ExpressionVisitor[str]):
-    def __init__(self, use_entity_aliases: bool = False):
+    def __init__(self, use_entity_aliases: bool = False, is_mql: bool = False):
```

Comment:
What can it be besides `mql`? In which cases would `is_mql` be False?

Reply:
This is used to encode both SnQL and MQL. In SnQL we use single quotes, `array(...)` and `tuple(...)`. In MQL we use double quotes and `[...]`.